index.html

```diff
@@ -0,0 +1,144 @@
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="program-cont">
+        <h3 class="program-title">Program Topics</h3>
+        <div class="program-position">
+          <section class="program-contents">
+            <div class="program-content-title">
+              <img src="src/assets/images/depositphotos_64999261-stock-illustration-abstract-runnig-woman-removebg-preview.png" class="program-content-img" alt="Healthy life icon">
+              <p class="program-contect-title-p">Healthy Life</p>
+              </div>
+            <div class="program-content-description">What are a healthy lifestyle and its importance of it. What will change in our life after this</div>
+          </section>
+          <section class="program-contents">
+            <div class="program-content-title">
+              <img src="src/assets/images/3576937.png" class="program-content-img" alt="Family icon">
+              <p class="program-contect-title-p">Family and Health.</p>
+              </div>
+            <div class="program-content-description">We will talk about, how a healthy lifestyle will affect your family.</div>
+          </section>
+          <section class="program-contents">
+            <div class="program-content-title">
+              <img src="src/assets/images/1403381.png" class="program-content-img" alt="Bad habits icon">
+              <p class="program-contect-title-p">Bad Habits</p>
+              </div>
+            <div class="program-content-description">We will touch on the topic of bad habits and how to deal with them, how your life will change after that</div>
+          </section>
+          <section class="program-contents">
+            <div class="program-content-title">
+              <img src="src/assets/images/child.png" class="program-content-img" alt="Children icon">
+              <p class="program-contect-title-p">Our Children</p>
+              </div>
+            <div class="program-content-description">The role of the volunteer movement in the formation of a healthy lifestyle among young people.</div>
+          </section>
+          <section class="program-contents">
+            <div class="program-content-title">
+              <img src="src/assets/images/4032473.png" class="program-content-img" alt="Medicine icon">
+              <p class="program-contect-title-p">Traditional Medicine</p>
+              </div>
+            <div class="program-content-description">The importance of traditional medicine and its role in shaping a healthy lifestyle.</div>
+          </section>
+        </div>
+        <div class="program-btn-cont">
+          <button class="program-btn">Join TIHL 2023</button>
+        </div>
+        <a href="" class="program-link">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <section class="speakers-section">
+        <div class="speakers-width">
+          <h3 class="speakers-title">Famous Speakers</h3>
+          <div class="speakers-cont">
+          </div>
```

Comment:
- [ ] On the mobile version, Kindly make all the speakers are not visible when the page load. some of them are hidden and we have the `SEE MORE` button feature once clicked the user can be able to see the remaining list of speakers, kindly add that. as It is in the provided template.
- [ ] Also make sure your speaker images are responsive. the size form the desktop version cannot be the same as those from the mobile version. Kindly consider fixing that.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198026128-478577eb-cea0-451e-8dda-d2cb7e088cfd.png)

Template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198026368-74327fca-3c6e-49a8-9513-63443e157733.png)